react/src/components/animations/Collapse/index.js

```diff
@@ -89,8 +89,8 @@ class Collapse extends React.Component {
 
     const collapseStyles = {
       [EXITED]: 'collapsed',
-      [EXITING]: 'expanding',
-      [ENTERING]: 'collapsing',
+      [EXITING]: 'collapsing',
+      [ENTERING]: 'expanding',
```

Comment:
Was this just wrong before, or what motivated this change? Asking because it will impact other components that use this as well I assume.